recipes/confu_json/all/conanfile.py

```diff
@@ -28,25 +26,31 @@ def _minimum_cpp_standard(self):
     @property
     def _minimum_compilers_version(self):
         return {
-            "Visual Studio": "15",
+            "Visual Studio": "17",
             "gcc": "10",
             "clang": "10",
         }
 
-    def validate(self):
+
+       
+    def configure(self):
+        if is_msvc(self) and Version(self.version) < "0.0.9":
+            raise ConanInvalidConfiguration(
+                "Visual Studio is not supported in versions before confu_json/0.0.9")
         if self.settings.compiler == "apple-clang":
             raise ConanInvalidConfiguration(
-                "apple-clang is not supported because of missing concept support")
+                "apple-clang is not supported. Pull request welcome")
```

Comment:
From Conan's perceptive it's a toolchain but I don't mind encourage people to contribute